client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -18,20 +18,21 @@ class TopCharts extends Component {
         </div>
     );
 
-    const charts = [topFree, topGrossing, trending, topPaid];
-
+    const { data } = this.props;    
+    if (data == null) {
```

Comment:
We can omit this since we only render TopCharts when App.js finds a TopCharts in the data.